### ISSUE TITLE
Legion Crusher achievement unobtainable

### ISSUE THREAD

**Issue by The-Moon-Itself** [BeeStation/BeeStation-Hornet#6024]
## Round ID:
34876 I think

## Reproduction:
When I killed legion with a crusher, I got the legion kill achievement, but not the legion crusher achievement. One note is that I killed it with a regular hit with the crusher instead of a mark detonation, but I'm not sure if that matters or not for this bug.

**Comment by PowerfulBacon** [comment]
Looking at the code, 60% of the boss damage must be done with the crusher and the crusher must be held when you perform the kill. It doesn't matter if the boss dies to the crusher or natural causes, as long as those 2 conditions are met.

**Comment by The-Moon-Itself** [comment]
The first time I did this and what prompted me to make this issue, I killed legion with crusher only. I also did it again yesterday with crusher only and I still didn't get the achievement. My best guess as to why its bugged is that I used the ash drake crusher trophy as well, which has AoE damage, but even if that doesn't count towards crusher damage, I don't think it would do more than 40% damage anyways though.
